lib/simulator-xcode-6.js

```diff
@@ -433,13 +433,18 @@ class SimulatorXcode6 extends EventEmitter {
    *   Equals to null by default, which keeps the current scale unchanged.
```

Comment:
would it be possible to move opts type description into a separate `typedef` block, like we do with all complex argument types?